sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -36,6 +42,18 @@ export interface EventData {
     };
 }
 
+// @public
+export class EventDataBatch {
+    // Warning: (ae-forgotten-export) The symbol "ConnectionContext" needs to be exported by the entry point index.d.ts
+    // 
+    // @internal
+    constructor(context: ConnectionContext, maxSizeInBytes: number, partitionKey: string);
+    currentSize: number;
```

Comment:
shouldn't this be just `size`? From outside in perspective `current` doesn't feel relevant.